tests/test_search.py

```diff
@@ -19,3 +25,34 @@ def test_empty_response(client) -> None:
 
     assert validator.validate(xml) is True, validator.error_log
     assert root.getchildren() == []
+
+
+def test_search(client, mocker):
+    es_host = 'http://libero-test-elasticsearch'
+    search.ELASTICSEARCH_HOSTS = [es_host]
+    es_search = mocker.patch('search.search.Elasticsearch.search')
+    es_search.return_value = {
+        'hits': {
+            'hits': [
+                {
+                    '_source': {
+                        'id': 'article1', 'service': 'scholarly-articles'
+                    }
+                }
+            ]
+        }
+    }
+
+    path = get_schema_file_path('item-list.rng')
+    xmlschema_doc = etree.parse(path)
+    validator = isoschematron.Schematron(xmlschema_doc)
+
+    response = client.get('/search?article1')
```

Comment:
currently `/search` is used as a `service-prefix` that is stripped out, so this would have to be implemented on `/`. Do we want to update the spec to give it an explicit resource e.g. `/items` or should it stay on `/`?

Reply:
I think this further identifies the need to plan and write up tasks more precisely as I was not aware of this.

Reply:
I think the prefix and the path inside the service are being confused. This path doesn't contain the prefix which is handled at the gateway.

Independently the spec has the proposal to rename the path `/items` (to avoid `/search/search` is the service prefix is also search). Though thinking about it, there isn't the same concern as with the content API as there's no sub paths, so this probably could just be `/`. But that's more for the spec.